torch/fx/experimental/symbolic_shapes.py

```diff
@@ -217,17 +217,13 @@ class FloorDiv(sympy.Function):
         2. Printing out the expression is nicer (compared to say, representing a//b as (a - a % b) / b)
         """
         nargs = (2,)
+        precedence = 49  # precedence of mul - 1  # noqa: F811
```

Comment:
Shouldn't floordiv have the same precedence as mul, per Python's operator precedence rules?

Reply:
That works, thanks. The printer handles commutativity conservatively.
Fixed.